### DISCUSSION TITLE
About "Code can be copied directly from the diff"

### DISCUSSION THREAD

**Discussion by AdmiralRazorbeard** [dandavison/delta#944] [Q&A] [answered]
Hi,

Sorry to bother you with this question but what does "Code can be copied directly from the diff" mean ?
I didn't find anything in the documentation. Does that mean I can copy code from the terminal and I will not have the lines numbers being copied ? If this is it, it doesn't seem to work.

Thanks

**Comment by dandavison** [comment, marked as answer]
Hi @AdmiralRazorbeard, what I was getting at there is that the traditional `+/-` markers that git uses are by default not present when using delta. However, you're right, if you use delta line numbers then those will get in the way of copy/pasting.

**Comment by AdmiralRazorbeard** [comment]
Thanks !

**Comment by raisedadead** [comment]
You can turn line-numbers off per invocation like so, if you are always having it on in your config.

```
git -c delta.line-numbers=false diff
```

**Comment by searleser97** [comment]
What about when using side-by-side=true ? Can we copy paste just the right or just the left side to clipboard ?

**Reply by dandavison** [reply]
I don't know how to (and I suppose it will be terminal emulator / tmux etc dependent), but I'd love to know of any tricks for doing things like this if there are any. I seem to remember that iTerm2 has a way to stop copying at tmux pane splits?
